ansible/roles/installation/tasks/main.yml

```diff
@@ -39,20 +39,12 @@
     POSTMARK_TRANSACTIONAL_STREAM: "{{ rails.postmark.transactional_stream }}"
     EMAIL_FROM_ADDRESS: "{{ rails.email_from_address }}"
     SECRET_KEY_BASE: "{{ rails.secret_key_base }}"
-    RAILS_INBOUND_EMAIL_PASSWORD: "{{ rails.inbound_email_password }}"
```

Comment:
This one needs to stay. It's a "magic" environment variable that is consumed by Rails itself.

https://guides.rubyonrails.org/action_mailbox_basics.html#postmark